runtime/runtime/src/lib.rs

```diff
@@ -1385,12 +1385,9 @@ impl Runtime {
         &self,
```

Comment:
Remove the following
- function `apply_migrations`
- `migration_data` in ApplyState
- `migration_flags` in ApplyState
- `migration_data` in NightshadeRuntime
- delete file chain/chain/src/runtime/migrations.rs
- mainnet_restored_receipts and mainnet_storage_usage_delta